sycl/test-e2e/USM/prefetch_exp.cpp

```diff
@@ -0,0 +1,113 @@
+//==-------- prefetch_exp.cpp - Experimental 2-way USM prefetch test -------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: %{build} -o %t1.out
+// RUN: %{run} %t1.out
+
+#include <sycl/detail/core.hpp>
+#include <sycl/ext/oneapi/experimental/enqueue_functions.hpp>
+#include <sycl/usm.hpp>
+
+using namespace sycl;
+
+static constexpr int count = 100;
```

Comment:
Variable names in LLVM should start with an upper case letter. Please apply to all the variables.

Reply:
Please, apply also to other files.